docs/openapiv2/apidocs.swagger.json

```diff
@@ -1046,44 +1052,65 @@
     },
     "AuthorizationModel": {
       "type": "object",
-      "example": {
-        "id": "01G5JAVJ41T49E9TT3SKVS7X1J",
-        "type_definitions": [
-          {
-            "type": "document",
-            "relations": {
-              "reader": {
-                "union": {
-                  "child": [
-                    {
-                      "this": {}
-                    },
-                    {
-                      "computedUserset": {
-                        "object": "",
-                        "relation": "writer"
-                      }
-                    }
-                  ]
-                }
-              },
-              "writer": {
-                "this": {}
-              }
-            }
-          }
-        ]
-      },
       "properties": {
         "id": {
           "type": "string",
           "example": "01G5JAVJ41T49E9TT3SKVS7X1J"
         },
         "type_definitions": {
           "type": "array",
+          "example": [
+            {
+              "type": "user"
+            },
+            {
+              "type": "document",
+              "relations": {
+                "reader": {
+                  "union": {
+                    "child": [
+                      {
+                        "this": {}
+                      },
+                      {
+                        "computedUserset": {
+                          "object": "",
+                          "relation": "writer"
+                        }
+                      }
+                    ]
+                  }
+                },
+                "writer": {
+                  "this": {}
+                }
+              },
+              "metadata": {
+                "relations": {
+                  "reader": {
+                    "directly_related_user_types": [
+                      {
+                        "type": "user"
+                      }
+                    ]
+                  },
+                  "writer": {
+                    "directly_related_user_types": [
+                      {
+                        "type": "user"
+                      }
+                    ]
+                  }
+                }
+              }
+            }
+          ],
           "items": {
             "$ref": "#/definitions/TypeDefinition"
           }
+        },
+        "schema_version": {
+          "type": "string"
```

Comment:
Valid one please

Reply:
Done in the validator. It's not the best, but unless we want to have versions like `SCHEMA_VERSION_ONE`, it's what we can do.